packages/design-system/src/components/ButtonsWrapper/style.scss

```diff
@@ -0,0 +1,11 @@
+.btns-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  margin-top: -8px;
+
+  & > * {
+    margin-right: 8px;
```

Comment:
is it fine to have always 8px margin on the right side? I know it's not much :)

Reply:
It should be zero after last element. I'll fix it